cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -268,8 +268,8 @@ function Set-EnvironmentVars {
     "KUBELET_CERT_PATH" = ${kube_env}['PKI_DIR'] + '\kubelet.crt'
     "KUBELET_KEY_PATH" = ${kube_env}['PKI_DIR'] + '\kubelet.key'
 
-    "CONTAINER_RUNTIME" = ${kube_env}['CONTAINER_RUNTIME']
-    "CONTAINER_RUNTIME_ENDPOINT" = ${kube_env}['CONTAINER_RUNTIME_ENDPOINT']
+    "WINDOWS_CONTAINER_RUNTIME" = ${kube_env}['WINDOWS_CONTAINER_RUNTIME']
```

Comment:
You don't need to rename the variables on the left side of these expressions - just change the right side.